ui/projects/spline-common/graph/src/components/control-panel/controls/overview-control/sg-overview-control.component.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ChangeDetectionStrategy, Component, EventEmitter, Output } from '@angular/core'
+
+import { OverviewTypeEnum } from '../../../../models/sg-overview-type.models'
+
+
+@Component({
+    selector: 'sg-overview-control',
+    templateUrl: './sg-overview-control.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SgOverviewControlComponent {
+    @Output() changeOverview$ = new EventEmitter<{ overviewType: OverviewTypeEnum }>()
+
+    overviewTypeEnum = OverviewTypeEnum
```

Comment:
```suggestion
    readonly overviewTypeEnum = OverviewTypeEnum
```
Generally, because we inclined to using a functional programming style in majority projects, we prefer enforcing immutability unless the opposite is clearly justified.